.buildkite/pipelines/pull-request/full-bwc.yml

```diff
@@ -0,0 +1,17 @@
+config:
+  allow-labels: test-full-bwc
+steps:
+  - group: bwc
+    steps:
+      - label: $BWC_VERSION / bwc
+        key: "full-bwc:$BWC_VERSION_SNAKE"
+        bwc_template: true
```

Comment:
Why is this used for the "full" BWC suites but not snapshots? Why not leverage matrix jobs for both?

Reply:
Because matrix jobs have a maximum of like 20 elements per matrix element, and there are way more BWC versions than that, so we just have to define each one as a step. ☹️

Reply:
🤦